tests/antithesis/test-template/robustness/common/path.go

```diff
@@ -34,8 +35,12 @@ const (
 	localetcd2 = "127.0.0.1:32379"
 	// used by default when running the client locally
 	defaultetcdLocalDataPath = "/tmp/etcddata%d"
-	localetcdDataPathEnv     = "ETCD_ROBUSTNESS_DATA_PATH"
+	localetcdDataPathEnv     = "ETCD_ROBUSTNESS_LOCAL_DATA_PATH"
 	localReportPath          = "report"
+
+	serverEndpointEnv = "ETCD_ROBUSTNESS_SERVER_ENDPOINTS"
+	dataPathEnv       = "ETCD_ROBUSTNESS_DATA_PATHS"
```

Comment:
What's the benefit of separating `ETCD_ROBUSTNESS_DATA_PATH` and `ETCD_ROBUSTNESS_LOCAL_DATA_PATH`? From binary point of view both are local.

Reply:
Will be reworking how the existing code currently handles the paths it uses as I [mentioned here ](https://github.com/etcd-io/etcd/pull/20668#issuecomment-3290221413) then.  Right now all the paths are baked into the code.